db/seeds.rb

```diff
@@ -43,18 +43,23 @@
   Category.find_or_create_by(name: "Programming languages")
 
   # create 350 users
-  350.times do |n|
+  46.times do |n|
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.<br>Inconsistent indentation detected.